WasteManagement/WasteContainer/doc/spec.md

```diff
@@ -215,7 +221,7 @@ to keep backwards compatibility with current FIWARE reference implementations.
     + Attribute metadata:
         + `dateUpdated`: Timestamp when the last update of the attribute happened.
             + Type: [DateTime](http://schema.org/DateTime)
-        + `TimeInstant` : Timestamp saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
+        + `TimeInstant` : [Timestamp](https://github.com/telefonicaid/iotagent-node-lib/blob/develop/README.md#TimeInstant) saved by FIWARE's IoT Agents. Note: This attribute has not been harmonized
```

Comment:
the problem is that we have the TimeInstant definition in develop, is that landing to master soon? @dmoran